performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md

```diff
@@ -0,0 +1,371 @@
+image, updating the Process status accordingly
+
+### classifyImageAsync
+```swift
+@available(iOS 15, tvOS 15, *) func classifyImageAsync() async
+```
+attempts to classify the Process image asynchronously and update the Process categories with the 
+results, measuring the classification task with a custom code trace and updating the Process status
+ accordingly
+
+### generateSaliencyMapAsync
+```swift
+@available(iOS 15, tvOS 15, *) func generateSaliencyMapAsync() async
+```
+attempts to generate the Process saliency map asynchronously, measuring the task with a custom code
+ trace and updating the Process status accordingly
+
+### uploadSaliencyMapAsync
+```swift
+@available(iOS 15, tvOS 15, *)
+func uploadSaliencyMapAsync(compressionQuality: CGFloat = 0.5) async
+```
+attempts to upload the Process saliency map asynchronously as `saliency_map.jpg` using 
+compressionQuality, updating the Process status accordingly
+
+### updateStatus
+```swift
+func updateStatus(to newStatus: ProcessStatus, updateUploadStatus: Bool = false)
+```
+updates the Process status on the main thread, possibly updating the status of the upload task
+
+### updateImage
+```swift
+func updateImage(to newImage: UIImage?)
+```
+updates the Process image on the main thread
+
+### updateSaliencyMap
+```swift
+func updateSaliencyMap(to newSaliencyMap: UIImage?)
+```
+updates the Process saliency map on the main thread
+
+### downloadImage
+```swift
+func downloadImage()
+```
+attempts to download the image from the Process site and set it as the new Process image, updating 
+the Process status accordingly
+
+### classifyImage
+```swift
+func classifyImage()
+```
+attempts to classify the Process image and update the Process categories with the results, 
+measuring the classification task with a custom code trace and updating the Process status 
+accordingly
+
+### generateSaliencyMap
+```swift
+​​func generateSaliencyMap()
+```
+attempts to generate the Process saliency map, measuring the task with a custom code trace and 
+updating the Process status accordingly
+
+### uploadSaliencyMap
+```swift
+func uploadSaliencyMap(compressionQuality: CGFloat = 0.5)
+```
+attempts to upload the Process saliency map as 'saliency_map.jpg’ using compressionQuality, 
+updating the Process status accordingly
+
+### makeTrace
+```swift
+func makeTrace(called name: String) -> Trace?
+```
+attempts to create and return a trace, setting its custom attributes of “precision” to the Process 
+precision and “platform” to one of “iOS” or “tvOS”
+
+## ProcessDerivatives
+
+### ProcessTask
+```swift
+enum ProcessTask: String {
+  case download = "Download"
+  case classify = "Classification"
+  case saliencyMap = "Saliency Map"
+  case upload = "Upload"
+}
+```
+tracks the different process tasks
+
+### ProcessStatus
+```swift
+enum ProcessStatus: Equatable {
+  case idle
+  case running(ProcessTask)
+  case failure(ProcessTask)
+  case success(ProcessTask)
+```
```

Comment:
nit: missing closing `}`

Reply:
That was intentional in that I wanted to imply that there was more to the enum (see `text` and `view` right after), but I do agree it is unclear. I'll add the closing `}` with some ellipses in-between.

Reply:
Done ✅